ddtrace/propagation/http.py

```diff
@@ -616,7 +616,8 @@ def _get_tracestate_values(ts):
             if list_mem.startswith("dd="):
                 # cut out dd= before turning into dict
                 list_mem = list_mem[3:]
-                dd = dict(item.split(":") for item in list_mem.split(";"))
+                # since tags can have a value with a :, we need to only split on the first instance of :
+                dd = dict(item.split(":", 1) for item in list_mem.split(";"))
```

Comment:
Are we making this change to address the following edge case: `s::2` -> {`s` : `:2`} ? 

Do we need to handle the case where `:` is in the key? ex: `s:_h:2` -> {`s:_h` : `:2`} ? Is this possible?

Reply:
Yes to the first question. 

To the second I'm almost positive the answer is no. Otherwise it would be impossible to determine what was the key and what was the value. E.g. [in the docs](https://docs.datadoghq.com/getting_started/tagging/#define-tags) we can see `env:staging:east` key is `env`, value is `staging:east`

Reply:
Makes sense. If we need to handle the second case then we should raise this issue to the team.